tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md

```diff
@@ -0,0 +1,444 @@
+    point your browser at that URL to view the running application.
+
+Congratulations! Your application is now up and running!
+
+## Scale and update your application
+
+You'll now explore a few of the basic features of Kubernetes for managing your
+running app.
+
+### Setting the replica count
+
+Initially your deployment runs a single instance of your application. You may
+add more replicas using the `kubectl scale` command. For example, to add two
+additional replicas (for a total of three), run:
+
+    kubectl scale deployment hello-web --replicas=3
+
+Once the additional replicas are running, you can see the list of three pods
+by running:
+
+    kubectl get pods
+
+Kubernetes automatically allocates your running pods on the virtual machines
+in your cluster. You may configure pods in your deployment with specific
+resource requirements such as memory and CPU. See the
+[Kubernetes documentation](https://kubernetes.io/docs/home/) for more details.
+
+### Updating your application
+
+After you make a change to your app, redeploying is just a matter of
+building a new image and pointing your deployment to it.
+
+1.  Make a change to the app. (For example, modify the front page template.)
+
+2.  Perform a new build with a new version tag "v2":
+
+        gcloud container builds submit --tag=gcr.io/${PROJECT_ID}/hello:v2 .
+
+    Now you have two builds stored in your project, `hello:v1` and `hello:v2`.
+    In general it's good practice to set the image tag for each build to a
+    unique build number. This will let you identify and deploy any build,
+    making updates and rollbacks easy.
+
+3.  Set the deployment to use the new image:
+
+        kubectl set image deployment/hello-web hello-web=gcr.io/${PROJECT_ID}/hello:v2
+
+    This will perform a "rolling" update of all the running pods.
+
+4.  You can "roll back" to the earlier build by calling `kubectl set image`
+    again, specifying the earlier build tag.
+
+        kubectl set image deployment/hello-web hello-web=gcr.io/${PROJECT_ID}/hello:v1
+
+**Note:** If a deployment gets "stuck" because an error in the image prevents
+it from starting successfuly, you can roll back by "undoing" the rollout. See
+the [Kubernetes deployment documentation](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+for more info.
+
+## Clean up
+
+After you've finished this tutorial, you can clean up the resources you created
+on Google Cloud Platform so you won't be billed for them in the future. To
+clean up the resources, you can delete your Container Engine resources, or
+delete the entire project.
+
+### Deleting Container Engine resources
+
+To delete your app from Container Engine, you must remove both the load
+balancer and the Container Engine cluster.
+
+1.  Delete the service which will deallocate the load balancer:
+
+        kubectl delete service hello-web
+
+2.  The load balancer will be deleted asynchronously. Wait for that process to
```

Comment:
Can you clarify what output the user should be looking for, or will it be obvious?

Reply:
I think it should be obvious, but I added a sentence.